### ISSUE TITLE
Draw time increase after #22579

### ISSUE THREAD

**Issue by Graywolfe813** [CleverRaven/Cataclysm-DDA#22739]
Game version: Any after build 7002

Operating system: Windows, Linux, Android

Tiles or curses: Tiles

Mods active: None

Draw time spiked after PR #22579. The effect is mitigated by the performance boost this provided to hardware rendering. It's extremely noticeable in software

Profiling indicates that game::draw_ter(bool) is taking roughly 12 times as long as it previously did.

**Comment by RedPine91** [comment]
Is this the culprit that is making craft times excessively long?  Normally, when I install a vehicle part, the process is nearly instant, but now I suffer 3 sec of IRL time for every 5 min game time each installation takes.  This has made interacting with vehicles nigh unbearable.

I parked the vehicle far from zombies, no hordes are on minimap, I have only 1 NPC, and there are only 6 cargo spaces full of loot, so this is the only thing I can think of that could be slowing the game down.
